test/blackbox-tests/test-cases/coq/github5532.t

```diff
@@ -0,0 +1,26 @@
+Reproducing test case for #5532.
+
+  $ cat >foo.v <<EOF
+  > Lemma dummy : True.
+  > Proof. idtac "A". Qed.
+  > EOF
+
+  $ cat >dune-project <<EOF
+  > (lang dune 3.1)
+  > (using coq 0.3)
+  > EOF
+
+  $ cat >dune <<EOF
+  > (coq.theory
+  >  (name basic))
+  > EOF
+
+  $ dune build
+  File "dune", line 1, characters 0-26:
+  1 | (coq.theory
+  2 |  (name basic))
```

Comment:
Why does `dune build` print the `dune` file on an error in the `.v` file? I would expect just the following error message:
```
  $ dune build
  A
  File "./foo.v", line 2, characters 18-22:
  Error:  (in proof dummy): Attempt to save an incomplete proof
  ```

Reply:
Yeah, that is indeed a bit weird. I guess that the point is to tell you from which theory the source file comes from? In that case there could probably be a better message to print. I also noticed that the line number information seems wrong for the dune file: line 1 does not have that many characters, the range should also encompass line 2.

Reply:
Yes, that's definitively bizarre, in particular, the `A` was produced by coqc, so the association to the stanza doesn't seem proper.

cc: @rgrinberg

Reply:
It's not really strange, it's the default for how dune outputs failures for any failed action. You might wonder why OCaml errors have a nicer output, well it's because dune is able to extra the "embeddable location" out of ocaml{c,dep,opt} commands. If dune could extract the locations out of coq commands, coq's errors would get the same treatment.

Reply:
What I mean is that `A` here is not an error, but coming from the stdout of Coq.

Reply:
I guess the OCaml compiler rarely does that, if ever. That could be considered "debug info".